.github/workflows/helm.yml

```diff
@@ -80,8 +76,7 @@ jobs:
         uses: helm/chart-releaser-action@v1.6.0
         env:
           CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
-          CR_RELEASE_NAME_TEMPLATE: "{{ .Version }}"
+          CR_RELEASE_NAME_TEMPLATE: "helm-{{ .Version }}"
         with:
           charts_dir: deploy
           skip_existing: true
-          packages_with_index: true
```

Comment:
you don't need this anymore?

Reply:
the flag didn't do what I thought it was doing. I thought this would allow us to avoid needing GHA releases for helm releases, but that is still needed, it just moves the artifacts into gh pages for private/enterprise repos